src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, useMemo } from "react";
+
+import { useConnectRelatedVenues } from "hooks/useConnectRelatedVenues";
+import { useVenueId } from "hooks/useVenueId";
+
+import "./ScheduleVenueDescription.scss";
+
+export const ScheduleVenueDescription: FC = () => {
+  const venueId = useVenueId();
+
+  const { parentVenue, currentVenue } = useConnectRelatedVenues({
+    venueId,
+  });
+
+  // @debt: ideally this would find the top most parent of parents and use those details
+  const scheduleVenue = useMemo(
+    () => (parentVenue ? parentVenue : currentVenue),
+    [parentVenue, currentVenue]
+  );
+
+  return (
+    <div className="ScheduleVenueDescription">
+      <div className="ScheduleVenueDescription__main">
+        <div
+          className="ScheduleVenueDescription__pic"
+          style={{ backgroundImage: `url(${scheduleVenue?.host?.icon})` }}
+        />
+        <div className="ScheduleVenueDescription__title">
+          <h2 className="ScheduleVenueDescription__name">
+            {scheduleVenue?.name}
+          </h2>
+          <h3 className="ScheduleVenueDescription__subtitle">
+            {scheduleVenue?.config?.landingPageConfig?.subtitle}
+          </h3>
+        </div>
+      </div>
+      <div className="ScheduleVenueDescription__desc">
+        <p>{scheduleVenue?.config?.landingPageConfig?.description}</p>
+      </div>
```

Comment:
Are these variables using the `.?` pattern just because `scheduleVenue` might not be defined? If so.. should we be showing some form of loading pattern while we wait for that variable to get defined? Or are we happy to render the structure here as potentially empty, and then fill in just those bits once it loads?

@mike-lvov thoughts?

Reply:
Also, are we using `h2`, `h3`, etc here in the semantically correct way, or just for their various styles? For accessibility reasons we need to make sure we use them appropriately. If it's just for their styling, there are other ways we can get the same effect.

Reply:
> Are these variables using the .? pattern just because scheduleVenue might not be defined? If so.. should we be showing some form of loading pattern while we wait for that variable to get defined? Or are we happy to render the structure here as potentially empty, and then fill in just those bits once it loads?

@mike-lvov @sunny-viktoryia Looks like this still needs input?

Reply:
created a default venue name constant in the `settings`: https://github.com/sparkletown/sparkle/pull/1302/commits/0b6bf111b31f1e972b1bfda52fc0c80e89d7f36d
I think for others we should be okay if they are empty.

screenshots:
we have all the information:
![image](https://user-images.githubusercontent.com/63313732/118548482-5a825d80-b728-11eb-8d69-5ad9ad83779c.png)

we have nothing:
![image](https://user-images.githubusercontent.com/63313732/118548510-62da9880-b728-11eb-97a1-5ed4193921e1.png)

Reply:
> created a default venue name constant in the settings: 0b6bf11

It looks like `scheduleVenue` is potentially undefined just because it may not have loaded yet, so not sure that having a default venue name like this is the right solution here.

I was more thinking that perhaps we should have something like `if (!venueSchedule) return <>Loading...</> or similar before the main JSX block.

Not sure if that's a good choice here though..

---

Currently you're passing `venueId` in to `ScheduleVenueDescription`, I wonder if it would make sense to pass `venue` instead? Where is this called from, and does that have `venue` in scope already?

Seemingly `src/components/organisms/NavBarSchedule/NavBarSchedule.tsx` which doesn't seem to currently have `venue` in scope (though could read it from `useConnectRelatedVenues` I think)

`src/components/organisms/NavBarSchedule/NavBarSchedule.tsx` is rendered in a bunch of places (sidenote: I would imagine we can probably remove it from all of the venue templates like `ArtPiece` and similar probably, since it's rendered from `NavBar`?), but most noticeably from `src/components/molecules/NavBar/NavBar.tsx`, which definitely has `venue` in scope.

It'd probably be a bit 'props drilling'y to string this through the various components.. so let's not do that.

---

@sunny-viktoryia Maybe just make the default label "Schedule" here?

---

[OOS] I'm honestly not sure that there is even a lot of value in having the current venue's details here. I expect the intent is probably that it will be the sovereign venue's details (as that is representative of the 'main event'; since the schedule applies to the whole event). This is probably something we should create a backlog issue for and followup with @sparkletown/product + @sparkletown/design in future.

Reply:
yeah, it was possible to remove NavBarSchedule: https://github.com/sparkletown/sparkle/pull/1302/commits/367c7e8b948dc58e9e7ffa10620e0010075640dc
moved default venue name from the settings: https://github.com/sparkletown/sparkle/pull/1302/commits/41bfff843417a7b10e7bfe90c7750ee4027118ff

Reply:
> moved default venue name from the settings: 41bfff8

From above:

> @sunny-viktoryia Maybe just make the default label "Schedule" here?

I would just do that inline, not extracted as a `const`

Reply:
did that in inline: https://github.com/sparkletown/sparkle/pull/1302/commits/61c8d31c7eecfeae9b1debbdc10b22f31a630afe

Reply:
We want `"Schedule"` not `"Sparkle"`

Reply:
done: https://github.com/sparkletown/sparkle/pull/1302/commits/eaea7348e4541095956568dd31f39dcd55161ee7